src/store/modules/nft.js

```diff
@@ -0,0 +1,48 @@
+/* eslint no-param-reassign: "off" */
+import Vue from 'vue';
+import { getNFTs } from '~/util/nft';
+import { getUserSellNFTClasses } from '@/util/api';
+
+const axios = require('axios');
+
+const NFT_SET_USER_CLASSID_LIST = 'NFT_SET_USER_CLASSID_LIST';
+
+const state = () => ({
+  userClassIdList: {},
+});
+
+const mutations = {
+  [NFT_SET_USER_CLASSID_LIST](state, { address, nfts }) {
+    Vue.set(state.userClassIdList, address, nfts);
+  },
+};
+
+const getters = {
+  NFTClassIdList: state => state.userClassIdList,
+  NFTClassIdsByAddress: state => address => state.userClassIdList[address],
```

Comment:
For consistency with state
Since this is a getter function, add prefix `get`
```suggestion
  getNFTClassIdListByAddress: state => address => state.userClassIdList[address],
```